pkg/workflow/controller/handlers/workflowTrigger/handler.go

```diff
@@ -0,0 +1,53 @@
+package workflowTrigger
+
+import (
+	log "github.com/sirupsen/logrus"
+
+	"github.com/caicloud/cyclone/pkg/apis/cyclone/v1alpha1"
+	"github.com/caicloud/cyclone/pkg/workflow/controller/handlers"
+)
+
+type Handler struct{}
+
+var (
+	// Check whether *Handler has implemented handlers.Interface interface.
+	_ handlers.Interface = (*Handler)(nil)
+)
+
+func (h *Handler) ObjectCreated(obj interface{}) {
+	// h.process(obj)
+	if wft := ToWorkflowTrigger(obj); wft != nil {
+		CreateCron(wft)
+	}
+}
+
+func (h *Handler) ObjectUpdated(obj interface{}) {
+	// h.process(obj)
+	if wft := ToWorkflowTrigger(obj); wft != nil {
+		UpdateCron(wft)
+	}
+}
+
+func (h *Handler) ObjectDeleted(obj interface{}) {
+	// h.process(obj)
+	if wft := ToWorkflowTrigger(obj); wft != nil {
```

Comment:
If `wft` is nil, there should be error handlong, at least error log is required. And I'd suggest to use return type `(wfr, err)` for `ToWorkflowTrigger`